### PR TITLE
PP-7871: Add smoke tests

### PR DESCRIPTION
**Pull request by adityapahuja**
This pull request has 6 commits.

1. The first commit adds smoke tests to test making card payments to ePDQ, Sandbox, Smartpay, Stripe and Worldpay.
2. The second commit adds a smoke test to test cancelling a card payment to Sandbox.
3. The third commit adds a product smoke test to test using a payment link to Sandbox gateway account.
4. The fourth commit updates all smoke test scripts to be able to retrieve secret values (e.g. API token, Public API URL address, etc.) needed for running smoke tests.
5. The fifth commit comments out smoke tests which are currently disabled or they don't have API tokens.
6. The sixth commit updates `README.md` on how to run smoke tests locally.

### CODE REVIEW

File: helpers/smokeTestHelpers.js [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 const log = LOCAL_SMOKE_TEST === 'true' ? require('../stubs/syntheticsLoggerStub/index.js') : require('SyntheticsLogger')
 const https = require('https')
+const AWS = require('aws-sdk')
 
-const headers = {
-  Authorization: `Bearer ${API_TOKEN}`,
-  'Content-Type': 'application/json'
-}
+const today = new Date()
+const thisMonthNextYear = new Date(today.getFullYear() + 1, today.getMonth(), today.getDate())
```
Comment (katstevens) [inline helpers/smokeTestHelpers.js:9]: I'm curious, what happens if this smoke test runs on 29th February in a leap year (2024)? Does it tick over to 1st March 2025? I guess we have a couple of years to find out...

**Review by katstevens — APPROVED** [review]
Looking good - let's see how it runs in the Canary 👍

File: helpers/smokeTestHelpers.js (hunk 1 above)
Comment (alexbishop1) [reply helpers/smokeTestHelpers.js:9]: Just tried it and it does become 1 March 2025.

Comment (adityapahuja) [reply helpers/smokeTestHelpers.js:9]: Yes, I ran the same code after setting the current date to 2024-02-29. 

```
Current date is 2024-02-29T00:00:00.000Z
Future date is 2025-03-01T00:00:00.000Z
```

**Review by katstevens — APPROVED** [review]

**Pull request merged**